jetstream/fox-guide-reader-mode-existing-user-part-2.toml

```
@@ -0,0 +1,31 @@
+[experiment.exposure_signal]
+window_end = 14
```

The exposure signal needs a SQL statement that determines whether a client has been exposed (for example, has seen a specific event)

```suggestion
[experiment]

[experiment.exposure_signal]
name = "todo"
friendly_name = "TODO"
description = "TODO"
select_expression = "event_method = 'view' AND event_object = 'on'" # ? todo
data_source = "reader_mode_events"
window_start = 0
window_end = 14
```